using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -56,7 +56,6 @@ Here are the key-value pairs in this event:
 | **Key or Key Type** | **Value Type** | **Optional** | **Description** |
 | :--- | :--- | :--- | :--- |
 | `mid (String)` | String | false | Auto Generated MID |
-| `vid (String)` | String | false | Blank if not present |
```

Comment:
Should we consider mentioning it as a deprecated shared state key in case anyone uses this value in a third party extension?